azure-pipelines.yml

```diff
@@ -668,7 +668,12 @@ jobs:
       brew uninstall azure-cli
       
       echo == Install azure-cli.rb formula ==
-      brew install --build-from-source $SYSTEM_ARTIFACTSDIRECTORY/homebrew/azure-cli.rb
+      # Need to create a dummy homebrew tap to install the formula
+      git config --global user.email "you@example.com"
+      git config --global user.name "Your Name"
+      brew tap-new dev/azure-cli
+      cp $SYSTEM_ARTIFACTSDIRECTORY/homebrew/azure-cli.rb $(brew --repository)/Library/Taps/dev/homebrew-azure-cli/Formula/
+      brew install --build-from-source dev/homebrew-azure-cli/azure-cli
```

Comment:
`--build-from-source` is required. Otherwise, brew tries to install azure-cli from core repo.
```
brew install dev/homebrew-azure-cli/azure-cli
==> Fetching downloads for: azure-cli
==> Downloading https://ghcr.io/v2/homebrew/core/azure-cli/manifests/2.77.0
==> Fetching dev/azure-cli/azure-cli
==> Downloading https://ghcr.io/v2/homebrew/core/azure-cli/blobs/sha256:3722168cdfb5ee06e816a2d9e6644feb1e59a8ebe9dfe43858ba6f285b1b4c02
==> Installing azure-cli from dev/azure-cli
==> Pouring azure-cli--2.77.0.sonoma.bottle.tar.gz
🍺  /usr/local/Cellar/azure-cli/2.77.0: 16,043 files, 329.2MB
==> No outdated dependents to upgrade!
```